### ISSUE TITLE
[MuonGun] I3MCEventHeader tweaks (Trac #1326)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1318]
Make sure r137166 didn't break anything.

Also:
1) The test can't find PROPOSAL or MMC

Neither PROPOSAL nor MMC found!
Skipping test

2) the extruded_polygon.py test fails on my laptop (Ubuntu 14.04)
AssertionError: 
Items are not equal:
 ACTUAL: -66.17291405457317
 DESIRED: -66.17291405457316

That's pretty close, but I'm wondering why this never fails on the bots.

Migrated from https://code.icecube.wisc.edu/ticket/1326
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "Make sure r137166 didn't break anything.\n\nAlso:\n1) The test can't find PROPOSAL or MMC\n\nNeither PROPOSAL nor MMC found!\nSkipping test\n\n2) the extruded_polygon.py test fails on my laptop (Ubuntu 14.04)\nAssertionError: \nItems are not equal:\n ACTUAL: -66.17291405457317\n DESIRED: -66.17291405457316\n\nThat's pretty close, but I'm wondering why this never fails on the bots.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "[MuonGun] I3MCEventHeader tweaks", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-09-02T03:25:29", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150907T18:42:24: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 137317:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137317"
> Make test_StaticSurfaceInjector actually do something, and loosen extruded polygon test. Closes #1318.
> ```


* jvansanten changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas commented:

> Milestone IceSim 4.2 deleted


* olivas changed milestone from "IceSim 4.2" to ""
